docs/build/micro-rdk/encoder/incremental.md

```diff
@@ -21,7 +21,7 @@ These two pins provide the phase outputs used to measure the speed and direction
 Navigate to the **CONFIGURE** tab of your machine's page in [the Viam app](https://app.viam.com).
 Click the **+** icon next to your machine part in the left-hand menu and select **Component**.
 Select the `encoder` type, then select the `incremental` model.
-Enter a name for your encoder and click **Create**.
+Enter a name or use the automatically suggested name for your encoder and click **Create**.
```

Comment:
This looks good but I think you could also shorten to just "the suggested name"!